objects/authentication_factor.json

```diff
@@ -0,0 +1,85 @@
+{
+  "caption": "Authentication Factor",
+  "description": "The authentication factor object provides information about the factor used to authenticate in an mfa request.",
+  "name": "authentication factor",
+  "extends": "object",
+  "attributes": {
+    "factor_device": {
+      "description": "Device used to complete the authentication request.",
+      "requirement": "recommended"
+    },
+    "factor_type": {
+      "enum": {
+        "sms": {
+          "caption": "SMS"
+        },
+        "security question": {
+          "caption": "Security Question"
+        },
+        "call": {
+          "caption": "Call"
+        },
+        "totp": {
+          "caption": "Time-based One-time Password"
+        },
+        "push": {
+          "caption": "Push Factor"
+        },
+        "token": {
+          "caption": "Token"
+        },
+        "Google authentication factor": {
```

Comment:
I imagine OCSF will probably want to leave specific vendors out of the schema.  Is Google's auth factor a unique device, or is it a type of something else (token? security key?)